source/MaterialXTest/MaterialXCore/Material.cpp

```diff
@@ -54,3 +58,143 @@ TEST_CASE("Material", "[material]")
     REQUIRE(instanceSpecColor->getDefaultValue()->asA<mx::Color3>() == mx::Color3(0.0f));
     REQUIRE(doc->validate());
 }
+
+TEST_CASE("Material Discovery", "[material]")
+{
+    mx::DocumentPtr doc = mx::createDocument();
+
+    mx::FileSearchPath searchPath;
+    const mx::FilePath currentPath = mx::FilePath::getCurrentPath();
+    searchPath.append(currentPath / mx::FilePath("libraries"));
+    searchPath.append(currentPath / mx::FilePath("resources/Materials/TestSuite"));
+
+    searchPath.append(mx::FilePath::getCurrentPath() / mx::FilePath("libraries"));
+    loadLibraries({ "targets", "stdlib", "pbrlib" }, searchPath, doc);
```

Comment:
What is the motivation for loading the standard libraries during material discovery?  Usually I think of that step only being required in code generation, or in validation of nodes against their nodedefs.

Reply:
Just copy and paste of test setup. Removed.